packages/component_code_gen/code_gen/generate_component_code.py

```diff
@@ -10,19 +10,16 @@
 
 
 def generate_code(app, prompt, templates, tries):
+    validate_inputs(app, prompt, templates, tries)
+    db = supabase_helpers.SupabaseConnector()
+    docs_meta = db.get_app_docs_meta(app)
+    auth_meta = db.get_app_auth_meta(app)
+    auth_example = f"Here's how authentication is done in {app}:\n{auth_meta['component_code_scaffold_raw']}"
```

Comment:
Generally I'd recommend two newlines separating these just for clarity (easier to read sections when you print out the full prompt for debugging)

Reply:
Thanks, added more newlines

Reply:
Can you also handle the case where `component_code_scaffold_raw` isn't present in auth_meta? When I was working with Danny we hit that case (for new apps that hadn't synced with Supabase yet).